### ISSUE TITLE
Changes to URL only movie.nfo are not captured

### ISSUE THREAD

**Issue by nomandera** [MediaBrowser/Emby#3321]
If a movie is added with a URL only movie.nfo Emby correctly uses the info to identify the metadata.

If however this URL was wrong and is subsequently corrected, Emby does not capture this change automatically.

Choosing to manually refresh metadata also does not udpate to the new movie.

Choosing the identify release option I will assume fixes the issue but it should be unnecessary.

I would expect normal behavior would be to trigger an automatic reidentification of the metadata if the url only nfo is changed or at the very least allow the manual refresh metadata to use the new info

**Comment by LukePulverenti** [comment]
It does capture the changed imdb id, issue is you go to refresh the item and the MovieDb id is the highest priority movie fetcher, and that is keyed off of MovieDb id which still has the old value.

**Comment by nomandera** [comment]
Is there any progress on this? I cannot see to find a slick way to refresh the data other than renaming a load of folders and then renaming them back (which is way too time consuming and wasteful).

Is there a simpler way I can force Emby en masse to rescan and use the URL only movie.nfo data provided?

It is worth noting that this is made worse by the fact that at some time in the current beta cycle https://github.com/MediaBrowser/Emby/issues/2559 cropped up again (seems fixed now) but it has left me with quite a few bad matches.

**Comment by LukePulverenti** [comment]
I haven't had a chance to come back to this yet, thanks.

**Comment by nomandera** [comment]
Can you suggest any interim way other than manually finding each bad match one by one and manually identifying it (since a fix is not in progress).

**Comment by nomandera** [comment]
Polite re-ask of the above question.

What is the correct way to recover from this bug without obliterating the database to start again and  losing all the watched information etc in the process.

I simply want Emby to refresh all movies but pay attention to the URL only movie.nfo this time.

Edit: still interested in this answer but on closer inspection I see that URL only movie.nfo seems broken again as well. Not always just some times. Is there somewhere we can engage more real time to hammer this out once and for all.

**Comment by LukePulverenti** [comment]
You're changing the contents of an existing nfo to point to an entirely different movie. Isn't this going to be a pretty rare scenario?

**Comment by nomandera** [comment]
There are two scenarios this now:

1. The nfo contents are wrong due to human error and when fixed (within the nfo) are not picked up by Emby
2. Emby ignores the nfo and tags the movie incorrectly (usually bug)

Scenario 1 is rare-ish but human error is human error and there should be a way to recover from it in Emby without having to artificially rename folders on disk to trigger a new movie index.

Ideally these nfo changes should be detected automatically but if it has to be manual I would expect the intuitive way would be for one the Refresh Metadata options to to trigger a new movie index.

Scenario 2 is predominantly driven by bugs in handling URL only movie.nfo and tvshow.nfo. Unfortunately I seem to be the one spotting these and reporting them so I am suffering the most. It was broken and fixed after I reported it (thank you) in last release cycle and it broke and you fixed it during the last beta cycle (without me reporting it). Now I think I have found a one possibly two new test cases  where it doesnt work as expected.

So tl;dr you should be able to force Emby to properly try again to identify something and forget all its previous assumptions without leaving Emby itself.

**Comment by LukePulverenti** [comment]
Resolved in the upcoming 4.8 server release.
